### PR TITLE
Use structs for logging instead of strings

### PR DESCRIPTION
**Pull request by rslota**

### CODE REVIEW

File: lib/sparrow/apns.ex [hunk 1]
```diff
@@ -293,23 +302,6 @@ defmodule Sparrow.APNS do
     })
   end
 
-  @spec notification_contains_title_or_body?(Sparrow.APNS.Notification.t()) ::
```
Comment (rslota) [inline lib/sparrow/apns.ex:296]: This was a dead code :)

File: lib/sparrow/apns.ex [hunk 2]
```diff
@@ -83,9 +83,10 @@ defmodule Sparrow.APNS do
     request = Request.new(headers, json_body, path, timeout)
 
     _ =
-      Logger.debug(fn ->
-        "action=push_apns_notification, request=#{inspect(request)}"
-      end)
+      Logger.debug("Sending APNS push notificarion",
```
Comment (leszke) [inline lib/sparrow/apns.ex:86]: `notification`

File: lib/sparrow/apns/token_bearer.ex [hunk 3]
```diff
@@ -42,9 +42,13 @@ defmodule Sparrow.APNS.TokenBearer do
     update_tokens(state)
 
     _ =
-      Logger.info(fn ->
-        "worker=apns_tokens_bearer, action=init, result=success"
-      end)
+      Logger.info("Sttarting TokenBearer",
```
Comment (leszke) [inline lib/sparrow/apns/token_bearer.ex:45]: `Starting`

File: lib/sparrow/apns/token_bearer.ex [hunk 4]
```diff
@@ -55,9 +59,13 @@ defmodule Sparrow.APNS.TokenBearer do
...
```
Comment (leszke) [inline lib/sparrow/apns/token_bearer.ex:62]: `Starting`

**Pull request merged**